cli/stubs/install_certs.cmd

```diff
@@ -1,37 +0,0 @@
-@echo off
```

Comment:
This script specifically trusts any certs that are in the folder. Because you are using a .pem file and not changing its extension to .crt it will not pick it up as is. The script needs to be made aware of .pem files instead of being deleted.

Reply:
I didn't think it was really necessary to have a process in place for this as it's a single certificate that only needs to be installed in Windows once and for the lifetime of the installation.

The .pem is just because that's the code I lifted from laravel/valet.

I'm not overly familiar with Github and the PR process so I will need to revisit this when I have some time.

Reply:
Its needed because WSL does not currently provide easy access to the linux file system. The ability to browse the linux file system is a feature that will probably be in the next major windows up though. To make it more convenient for the windows users who are the target of this application it is best to export the CA to the windows file system.

Reply:
If I copy the `.pem` to `/mnt/c/tool/valet-wsl` as a .crt, it could be opened directly in Windows to be installed in the certificate store of choice. Doesn't that negate the need for a script to install certs from the directory?

Reply:
The script is just an extra convenience. It is set up to automatically install the crt to the correct store for it to be trusted root. This removed the need for a user to have knowledge of how a CA or crt works and where it should be placed to make this system work.